Procfile

```diff
@@ -1 +1,3 @@
 compute_actions: sidekiq -c 1 -r ./lib/lab_manager/sidekiq/boot.rb -q compute_actions
+scheduler: bundle exec ./bin/scheduler
+web: bundle exec unicorn -p 21000
```

Comment:
do not set port directly, see Twelve Factors article and particularly https://devcenter.heroku.com/articles/procfile for port specification